bin/utils/descriptions/fix_descriptions.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
+cd "$(readlink -e "$(dirname "$0")")"
+
+# This script uses PhantomJS to update the descriptions with the DOM generated by WebKit.
+# It thus replace seriously broken HTML by what is actually rendered in the browser.
+
+# Use like this:
+#   ./fix_descriptions.sh > update_descriptions.sql
+# Read what gets written on stderr, and eventually perform manual adjustments then:
+#   psql < update_descriptions.sql
+
+# PhantomJS binary to use
+PHANTOMJS="/opt/phantomjs/bin/phantomjs"
```

Comment:
would be nice if this could be set on the CLI or just used it from PATH - like this you have to edit the script if you don't have it in `/opt/phantomjs`